pybossa/cache/projects.py

```diff
@@ -107,7 +107,7 @@ def browse_tasks(project_id, args, filter_user_prefs=False, user_id=None):
         # for worker-view, validate worker_filter and compute preference score
         if filter_user_prefs:
             if not user_meet_task_requirement(row.id, w_filter, user_profile):
-                # if the user is not qualified for the task, skip
+                # exclude unqualified tasks for the user
```

Comment:
I would suggest saying something like "exclude tasks for which the user is unqualified"

Reply:
like the rephrasing.